trello_integration/trello_integration.php

```diff
@@ -0,0 +1,145 @@
+<?php
+
+// Get the environment; we will post a new comment to Trello each time
+// a commit appears on a new branch on Pantheon.
+$env = $_ENV['PANTHEON_ENVIRONMENT'];
+
+// Do not watch test or live, though.
+if (($env == 'live') || ($env == 'test')) {
```

Comment:
In the case of Trello, these commits never set any sort of status on the Trello card, right? In this instance, then, it makes sense to track activity on the card in every multidev that the commit moves through, as the current implementation does.